tests/functional/ipset_controller_test.go

```diff
@@ -312,6 +314,72 @@ var _ = Describe("IPSet controller", func() {
 		})
 	})
 
+	When("an IPSet with multiple networks gets created", func() {
+		var netSpecs []networkv1.Network
+		var networks []networkv1.IPSetNetwork
+		BeforeEach(func() {
+			netSpecs = []networkv1.Network{}
+			netSpecs = append(netSpecs, GetNetSpec(net1, GetSubnet1(subnet1)))
+			netSpecs = append(netSpecs, GetNetSpec(net2, GetSubnet2(subnet1)))
+			netSpecs = append(netSpecs, GetNetSpec(net3, GetSubnet3(subnet1)))
```

Comment:
i am not sure if the netspecs are required by the way.

i assume  yes as all the other test create both

i assume i need to create the network specification to then be able to assign IP from those network to the ipsets.

so i think this is required but just said i would call this out incase i can simplfy